src/cli/app.rs

```diff
@@ -73,6 +73,12 @@ fn wrap_with(app: App<'static, 'static>) -> App<'static, 'static> {
             .number_of_values(5)
             .multiple(true),
     )
+    .arg(
+        Arg::with_name(OperationId::Threshold.as_str())
+            .help("Operation: convert the image into only black and white")
```

Comment:
I would suggest to change the description to something along the lines of "Operation: Threshold the input image using Otsu's method"

Reply:
Oh OK sorry for this since I'm Not a native English speaker 😅